app/controllers/comments_controller.rb

```diff
@@ -20,4 +21,12 @@ def comment_params
     params.require(:comment).permit(:text).merge(user_id: current_user.id)
   end
   
+  def correct_user
+    @comment = Comment.find(params[:id])
+    @image = @comment.image
+    if @comment.user != current_user
```

Comment:
You'll want to make this whole method private as it is only used within the controller.

Likewise, since this is just a private method we don't need these to be instance variables (`@`). Instance variables in controllers are used to make those variables available to the view, but that is not needed with this method. Again, this will work fine, but its good to be explicit about these sort of things to make your work clear.

Lastly, a little thought experiment for you. What happens if the user is not logged in...?